flake.nix

```diff
@@ -1,7 +1,8 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
-    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
+    # nixpkgs - We use nixpkgs as input of `flakebox`, as it locks things like toolchains,
+    #           in versions that are actually tested in flakebox's CI to cross-compile things
+    #           well. This also saves us download and Nix evaluation time.
```

Comment:
This comment is a bit confusing, by reading the code I get what it means though. We are using the `nixpkgs` used by `flakebox, so we don't need it listed here (kind of the other way around than specifying that falkebox follows our `nixpkgs`).